src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -252,6 +265,26 @@ export default memo(function LoadoutBuilder({
     [statConstraints, resultSets]
   );
 
+  useEffect(() => hideItemPicker(), [selectedStore.classType]);
+
+  const chooseItem =
```

Comment:
I found the name confusing until I figured out what it was doing, maybe `chooseItemFromPicker`?

Reply:
Sure - it was always named that though